lib/govuk_ab_testing/minitest_helpers.rb

```diff
@@ -0,0 +1,29 @@
+module GovukAbTesting
+  module MinitestHelpers
+    def with_variant(args)
+      ab_test_name, variant = args.first
+
+      ab_test = GovukAbTesting::AbTest.new(ab_test_name)
+
+      previous_variant = @request.headers[ab_test.request_header]
+      @request.headers[ab_test.request_header] = variant
+
+      yield
+
+      @request.headers[ab_test.request_header] = previous_variant
+    end
+
+    def assert_ab_test_rendered(ab_test_name)
+      requested_variant = GovukAbTesting::RequestedVariant.new(ab_test_name, @request)
+
+      assert_equal requested_variant.response_header, response.headers['Vary']
```

Comment:
I don't think `RequestedVariant` delegates `response_header` to `AbTest` anymore? Does this actually work?